pkg/controller/plan/adapter/ovirt/client.go

```diff
@@ -73,6 +81,39 @@ func (r *Client) RemoveSnapshots(vmRef ref.Ref, precopies []planapi.Precopy) (er
 	return
 }
 
+//
+// Check if a snapshot is ready to transfer, to avoid importer restarts.
+func (r *Client) CheckSnapshotReady(vmRef ref.Ref, snapshot string) (ready bool, err error) {
+	correlationID, err := r.getSnapshotCorrelationID(vmRef, &snapshot)
+	if err != nil {
+		err = liberr.Wrap(err)
+		return
+	}
+	jobs, err := r.getJobs(correlationID)
+	if err != nil {
+		err = liberr.Wrap(err)
+		return
+	}
+	if len(jobs) > 1 {
+		err = liberr.New("Multiple jobs found for correlation ID %s", correlationID)
```

Comment:
I wouldn't make this an error, for snapshots there is one job, but it is not unlikely to have multiple jobs for the same correlation id

Reply:
Okay, I removed this check.